src/main/kotlin/com/ampnet/projectservice/controller/PublicOrganizationController.kt

```diff
@@ -34,4 +42,17 @@ class PublicOrganizationController(private val organizationService: Organization
         }
         return ResponseEntity.notFound().build()
     }
+
+    @GetMapping("/public/organization/{uuid}/members")
+    fun getOrganizationMembers(
+        @PathVariable("uuid") uuid: UUID
+    ): ResponseEntity<OrganizationMembershipsResponse> {
+        logger.debug { "Received request to get members for organization: $uuid" }
+        val members = organizationMembershipService.getOrganizationMemberships(uuid)
+        val users = userService.getUsers(members.map { it.userUuid })
+        val membersResponse = members.map {
+            OrganizationMembershipResponse(it, users.firstOrNull { user -> user.uuid == it.userUuid.toString() })
```

Comment:
Are you sure that this response will return data as defined in task? https://dev.azure.com/ampnetio/AMPnet%20Crowdfunding/_workitems/edit/674

User email and uuid should not be visible from public route